README.md

```diff
@@ -57,7 +57,7 @@ For more detailed information, see the [White Paper](https://www.iresty.com/down
 - **High performance**: The single-core QPS reaches 24k with an average delay of less than 0.6 milliseconds.
 - **Anti-ReDoS(Regular expression Denial of Service)**
 - **IP Whitelist/Blacklist**
-- **IdP**: Support external authentication services, such as Auth0, okta, etc., users can use this to connect to Oauth2.0 and other authentication methods.
+- **IdP**: Support external authentication services, such as Auth0, okta, Authing, etc., users can use this to connect to Oauth2.0 and other authentication methods.
```

Comment:
Could you help modify `okta` to `Okta` just like [this PR](https://github.com/iresty/apisix/pull/742) does?

Reply:
Modifed